test/controllers/update/overseas.entity.query.controller.spec.ts

```diff
@@ -88,5 +89,22 @@ describe("OVERSEAS ENTITY QUERY controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test('renders the OVERSEAS_ENTITY_QUERY_PAGE page with both error messages', async () => {
+      const resp = await request(app)
+        .post(config.OVERSEAS_ENTITY_QUERY_URL)
+        .send({ oe_number: '' });
```

Comment:
should not be spaces ```{oe_number: ''} ```

Reply:
@dgolubevs , lint:fix is configured to add spaces and can see the same format with spaces being used in other files also (ex. secure.register.filter.controller.spec.ts)